### ISSUE TITLE
Multiple issues while adding new product

### ISSUE THREAD

**Issue by RawPhunky** [magento/magento2#1734]
How stable is Magento2? I keep getting errors and page hangs almost everywhere I go in Admin panel... Tried both Master and Dev branches.

Here is An attempt to add Product on Clean Master branch install running on xampp Windows.

Go to Products/Catalog - Page does not load properly:
![1](https://cloud.githubusercontent.com/assets/2244332/9452141/3f227ef6-4abb-11e5-81e3-3dc3b26988fb.png)

Close Chrome Tab, reopen admin panel.
Go to Products/Catalog - Again Page does not load properly:
![2](https://cloud.githubusercontent.com/assets/2244332/9452139/3f20cbd8-4abb-11e5-98b9-39712aa2dc1f.png)

Do it 3rd time (close / reopen / go to catalog) Now Catalog page loads properly
![3](https://cloud.githubusercontent.com/assets/2244332/9452137/3f1f8aac-4abb-11e5-843d-1ccdd666210f.png)

Click on Add Product - You get error and nothing happens
![4](https://cloud.githubusercontent.com/assets/2244332/9452140/3f22373e-4abb-11e5-9c89-8dbd29b6e46e.png)

Close Chrome Tab, reopen admin panel. Go to Products/Catalog and click Add Product again you will finally enter Product creation page.
Try to create product, click save - Page hangs with warning
![5](https://cloud.githubusercontent.com/assets/2244332/9452138/3f205be4-4abb-11e5-979c-ad4219d8700e.png)

Issue might be related to: https://github.com/magento/magento2/issues/1725

**Comment by mazhalai** [comment]
Thank you @RawPhunky for your report. We have created MAGETWO-42232 to track a fix for this issue.

**Comment by mazhalai** [comment]
@RawPhunky can you please check your apache logs and see if there are any errors in that?

**Comment by bassdr** [comment]
I am having similar issues, and it seems to be related with javascript files not found.
Note that I chose French Canada at installation time, and it seems like it tries to open a file in the localization folder (pub/static/adminhtml/Magento/backend/fr_CA/tiny_mce/tiny_mce_src.js), the file is 
at lib/web/tiny_mce/tiny_mce_src.js

I am not using apache, but nginx, I carefully followed the nginx.conf.sample from the repo:

`==> /var/log/nginx/localhost.error_log <==
2015/09/30 02:36:00 [error] 10443#0: *187 open() "/var/www/bassdr.servehttp.com/htdocs/magento2/pub/static/adminhtml/Magento/backend/fr_CA/tiny_mce/tiny_mce_src.js" failed (2: No such file or directory), client: 127.0.0.1, server: bassdr.servehttp.com, request: "GET /magento2/pub/static/adminhtml/Magento/backend/fr_CA/tiny_mce/tiny_mce_src.js HTTP/1.1", host: "bassdr.servehttp.com:8080", referrer: "http://bassdr.servehttp.com:8080/magento2/admin/catalog/product/new/set/4/type/simple/key/a785d5a15c57f98844a107c7606681e5e41378eb06daf29c905481b5d84f3d92/"`

Also, I found this error:
`Uncaught Error: Script error for: Magento_Catalog/js/product-gallery/require.js:141
http://requirejs.org/docs/errors.html#scripterror`

Hope this helps!

**Comment by antonkril** [comment]
Did you run static files materialization? Magento 2 stores static files in module and theme folders and copies them to pub/static folder on request in dev mode. So if on one request the browser tries to load a lot of new static files, some of these requests will fall off by timeout (especially on windows systems). Reloading page usually fixes the problem. But there is also view file materialization command in bin/magento that will materialize all static files at once.

**Comment by bassdr** [comment]
Hi antonkril,
Thank you very much for your quick reply. I was able to fix the problem. If someone has the same issue, here is what I did:

First thing I realized when running bin/magento script, my .../magento/etc folder was not writable by my nginx user, that is the first thing I fixed (will care about security later). After that, from magento's bin folder, I've run: 
* `sudo -u nginx ./magento maintenance:enable`
* `sudo -u nginx ./magento setup:upgrade`
* `sudo -u nginx ./magento setup:static-content:deploy en_US en_CA fr_CA fr_FR`
* `sudo -u nginx ./magento maintenance:disable`

I kept having this error when it tries to process frontend -> Magento/luma: `PHP Fatal error:  Allowed memory size of 134217728 bytes exhausted (tried to allocate 262144 bytes) in /var/www/bassdr.servehttp.com/htdocs/magento2/vendor/oyejorge/less.php/lib/Less/Output.php on line 27`. It seems like the en_US localization works fine, but the fr_CA and fr_FR fails allocating that much memory.

I do have the recommended settings found the php.ini.sample, could it be a bug only in fr_CA localization? Anyhow, I changed `memory_limit = 64M` to `memory_limit = 512M`, 128M is still insufficient it seems...
Also, running ./magento will use php-cli instead of php-fpm used by the server, so I had to copy all changes to both to /etc/php/cli-php5.6/php.ini AND /etc/php/fpm-php5.6/php.ini for that reason

Maybe this will help someone having similar issues...
